docs/docs/migration/migration-from-v5-to-v6.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Migrating from v5 to v6
+sidebar_position: 1
+---
+
+# Migrating from v5 to v6
+
+## API
+
+### JobStatus
+
+The `JobStatus` enum was renamed to `KubernetesJobStatus`.
+
+### User-Permissions
+
+1. _Prerequisites_: Manage or sync allowed users in project (not covered here)
+
+2. Remove custom `CurrentUser` and `CurrentUserLoader`
+
+    ```diff
+    - declare module "@comet/cms-api" {
+    -     interface CurrentUserInterface {
+    -         ...
+    -     }
+    - }
+
+    - @ObjectType()
+    - export class CurrentUser implements CurrentUserInterface {
+    -     ...
+    - }
+
+    - export class CurrentUserLoader implements CurrentUserLoaderInterface {
+    -     ...
+    - }
+    ```
+
+    Also remove usage
+
+    ```diff
+      createAuthProxyJwtStrategy({
+          jwksUri: config.auth.idpJwksUri,
+    -     currentUserLoader: new CurrentUserLoader(),
+      }),
+    ```
+
+    Change imports of removed classes
+
+    ```diff
+    - import { CurrentUser } from "@src/auth/current-user";
+    - import { CurrentUser } from "@comet/cms-api";
+    ```
+
+    It shouldn't be necessary to overwrite these classes anymore, however, if you really need it, provide the CurrentUserLoader with `CURRENT_USER_LOADER`
+
+3. Create interface for `availablePermissions` similar to the already existing interface `interface ContentScope`
+
+    ```diff
```

Comment:
I would not use diff for all new files, because the green text is hard to read:

<img width="983" alt="image" src="https://github.com/vivid-planet/comet/assets/48853629/0bf5c70e-c22a-43f7-8ec5-b5417cec5ba6">

Reply:
I replaced the completely green diff blocks with ts/tsx blocks